### PR TITLE
Handle lack of activities + add activities for tests to use

### PR DESCRIPTION
**Pull request by sczerwinski-wikia**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java [hunk 1]
```diff
@@ -129,15 +127,20 @@ public Boolean doesLastNRecentBlogActivitiesContain(int n, String blogPostConten
...
+   * @return list of activities
    */
   public List<Activity> getActivities(int numberOfActivities) {
 
     List<Activity> activityList = new ArrayList<>();
     //numberOfActivitiesToFetch set to max number of possible elements to fetch
-    int numberOfActivitiesToFetch = numberOfActivities > activityWebElementList.size()
-            ? activityWebElementList.size() : numberOfActivities;
+    int numberOfActivitiesToFetch;
+    if (activityWebElementList.size()>0){
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java]: MAJOR Use isEmpty() to check whether the collection is empty or not. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1155)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 10 issues
* ![MAJOR][MAJOR] 6 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 9 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L78](https://github.com/Wikia/selenium-tests/blob/ba07b72443b9b7993a0f6038d0a93d6e9fb23008/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L78): Replace this usage of System.out or System.err by a logger. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L79](https://github.com/Wikia/selenium-tests/blob/ba07b72443b9b7993a0f6038d0a93d6e9fb23008/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L79): Replace this usage of System.out or System.err by a logger. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L80](https://github.com/Wikia/selenium-tests/blob/ba07b72443b9b7993a0f6038d0a93d6e9fb23008/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L80): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L96](https://github.com/Wikia/selenium-tests/blob/ba07b72443b9b7993a0f6038d0a93d6e9fb23008/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L96): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L114](https://github.com/Wikia/selenium-tests/blob/ba07b72443b9b7993a0f6038d0a93d6e9fb23008/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L114): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L85](https://github.com/Wikia/selenium-tests/blob/ba07b72443b9b7993a0f6038d0a93d6e9fb23008/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L85): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L102](https://github.com/Wikia/selenium-tests/blob/ba07b72443b9b7993a0f6038d0a93d6e9fb23008/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L102): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L120](https://github.com/Wikia/selenium-tests/blob/ba07b72443b9b7993a0f6038d0a93d6e9fb23008/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L120): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [WikiActivityTests.java#L103](https://github.com/Wikia/selenium-tests/blob/ba07b72443b9b7993a0f6038d0a93d6e9fb23008/src/test/java/com/wikia/webdriver/testcases/activityfeedstests/WikiActivityTests.java#L103): Define a constant instead of duplicating this literal "content" 4 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L78](https://github.com/Wikia/selenium-tests/blob/444772cffbbb2c24b9ba0e3f8a1cbfda41eb9191/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L78): Replace this usage of System.out or System.err by a logger. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L79](https://github.com/Wikia/selenium-tests/blob/444772cffbbb2c24b9ba0e3f8a1cbfda41eb9191/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L79): Replace this usage of System.out or System.err by a logger. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L80](https://github.com/Wikia/selenium-tests/blob/444772cffbbb2c24b9ba0e3f8a1cbfda41eb9191/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L80): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L96](https://github.com/Wikia/selenium-tests/blob/444772cffbbb2c24b9ba0e3f8a1cbfda41eb9191/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L96): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L114](https://github.com/Wikia/selenium-tests/blob/444772cffbbb2c24b9ba0e3f8a1cbfda41eb9191/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L114): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L85](https://github.com/Wikia/selenium-tests/blob/444772cffbbb2c24b9ba0e3f8a1cbfda41eb9191/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L85): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L102](https://github.com/Wikia/selenium-tests/blob/444772cffbbb2c24b9ba0e3f8a1cbfda41eb9191/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L102): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L120](https://github.com/Wikia/selenium-tests/blob/444772cffbbb2c24b9ba0e3f8a1cbfda41eb9191/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L120): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [WikiActivityTests.java#L103](https://github.com/Wikia/selenium-tests/blob/444772cffbbb2c24b9ba0e3f8a1cbfda41eb9191/src/test/java/com/wikia/webdriver/testcases/activityfeedstests/WikiActivityTests.java#L103): Define a constant instead of duplicating this literal "content" 4 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java [hunk 2]
```diff
@@ -6,9 +6,7 @@
 import com.wikia.webdriver.pageobjectsfactory.componentobject.activity.Activity;
 import com.wikia.webdriver.pageobjectsfactory.componentobject.activity.ActivityPageFactory;
 import com.wikia.webdriver.pageobjectsfactory.componentobject.activity.creators.*;
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
+import org.openqa.selenium.*;
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java]: Get rid of "import all"

**Comment by WikiaTech** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L78](https://github.com/Wikia/selenium-tests/blob/3c8dc43001402429b983c6a9d8da1b64d5c623e7/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L78): Replace this usage of System.out or System.err by a logger. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L79](https://github.com/Wikia/selenium-tests/blob/3c8dc43001402429b983c6a9d8da1b64d5c623e7/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L79): Replace this usage of System.out or System.err by a logger. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L80](https://github.com/Wikia/selenium-tests/blob/3c8dc43001402429b983c6a9d8da1b64d5c623e7/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L80): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L96](https://github.com/Wikia/selenium-tests/blob/3c8dc43001402429b983c6a9d8da1b64d5c623e7/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L96): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L114](https://github.com/Wikia/selenium-tests/blob/3c8dc43001402429b983c6a9d8da1b64d5c623e7/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L114): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L85](https://github.com/Wikia/selenium-tests/blob/3c8dc43001402429b983c6a9d8da1b64d5c623e7/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L85): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L102](https://github.com/Wikia/selenium-tests/blob/3c8dc43001402429b983c6a9d8da1b64d5c623e7/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L102): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L120](https://github.com/Wikia/selenium-tests/blob/3c8dc43001402429b983c6a9d8da1b64d5c623e7/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L120): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [WikiActivityTests.java#L103](https://github.com/Wikia/selenium-tests/blob/3c8dc43001402429b983c6a9d8da1b64d5c623e7/src/test/java/com/wikia/webdriver/testcases/activityfeedstests/WikiActivityTests.java#L103): Define a constant instead of duplicating this literal "content" 4 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L81](https://github.com/Wikia/selenium-tests/blob/131394a2f8cdeae54f5af3d942cf9d1d2aff0281/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L81): Replace this usage of System.out or System.err by a logger. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L82](https://github.com/Wikia/selenium-tests/blob/131394a2f8cdeae54f5af3d942cf9d1d2aff0281/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L82): Replace this usage of System.out or System.err by a logger. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L83](https://github.com/Wikia/selenium-tests/blob/131394a2f8cdeae54f5af3d942cf9d1d2aff0281/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L83): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L99](https://github.com/Wikia/selenium-tests/blob/131394a2f8cdeae54f5af3d942cf9d1d2aff0281/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L99): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L117](https://github.com/Wikia/selenium-tests/blob/131394a2f8cdeae54f5af3d942cf9d1d2aff0281/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L117): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L88](https://github.com/Wikia/selenium-tests/blob/131394a2f8cdeae54f5af3d942cf9d1d2aff0281/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L88): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L105](https://github.com/Wikia/selenium-tests/blob/131394a2f8cdeae54f5af3d942cf9d1d2aff0281/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L105): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L123](https://github.com/Wikia/selenium-tests/blob/131394a2f8cdeae54f5af3d942cf9d1d2aff0281/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L123): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [WikiActivityTests.java#L103](https://github.com/Wikia/selenium-tests/blob/131394a2f8cdeae54f5af3d942cf9d1d2aff0281/src/test/java/com/wikia/webdriver/testcases/activityfeedstests/WikiActivityTests.java#L103): Define a constant instead of duplicating this literal "content" 4 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L81](https://github.com/Wikia/selenium-tests/blob/1157160be2d884413423a5ddbff031a5f3e328b9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L81): Replace this usage of System.out or System.err by a logger. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L82](https://github.com/Wikia/selenium-tests/blob/1157160be2d884413423a5ddbff031a5f3e328b9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L82): Replace this usage of System.out or System.err by a logger. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L83](https://github.com/Wikia/selenium-tests/blob/1157160be2d884413423a5ddbff031a5f3e328b9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L83): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L99](https://github.com/Wikia/selenium-tests/blob/1157160be2d884413423a5ddbff031a5f3e328b9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L99): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MAJOR][MAJOR] [SpecialWikiActivityPageObject.java#L117](https://github.com/Wikia/selenium-tests/blob/1157160be2d884413423a5ddbff031a5f3e328b9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L117): Introduce a new variable instead of reusing the parameter "n". [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L88](https://github.com/Wikia/selenium-tests/blob/1157160be2d884413423a5ddbff031a5f3e328b9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L88): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L105](https://github.com/Wikia/selenium-tests/blob/1157160be2d884413423a5ddbff031a5f3e328b9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L105): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [SpecialWikiActivityPageObject.java#L123](https://github.com/Wikia/selenium-tests/blob/1157160be2d884413423a5ddbff031a5f3e328b9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java#L123): Remove the literal "true" boolean value. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1125)
1. ![MINOR][MINOR] [WikiActivityTests.java#L103](https://github.com/Wikia/selenium-tests/blob/1157160be2d884413423a5ddbff031a5f3e328b9/src/test/java/com/wikia/webdriver/testcases/activityfeedstests/WikiActivityTests.java#L103): Define a constant instead of duplicating this literal "content" 4 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java (hunk 2 above)
Comment (sczerwinski-wikia) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java]: Done

**Pull request merged**